app/src/main/res/layout/profile_list_profile_view.xml

```diff
@@ -36,6 +36,7 @@
       android:layout_width="0dp"
       android:layout_height="wrap_content"
       android:layout_marginStart="20dp"
+      android:layout_marginBottom="10dp"
```

Comment:
Why was this value added ? It was not available earlier.

Reply:
The profile image and profile name were not in symmetric order. So I added 10dp of marginBottom to the profile name textview.

Reply:
The profile image and profile name were not in symmetric order. So I added 10dp of marginBottom to the profile name textview.

Reply:
But that should be filed as a separate issue and then solved. As this PR is focused on merging of files only and therefore this change can create confusion. Also we generally do not write random values in xml file, we check them from our design files and from that we get the correct value and use that.

Reply:
Thanks @rt4914  for guiding, I will make the required changes.